tracer/src/Datadog.Trace/Telemetry/Collectors/ConfigurationTelemetryCollector.cs

```diff
@@ -132,6 +133,8 @@ public ICollection<TelemetryValue> GetConfigurationData()
                 new(ConfigTelemetryData.FullTrustAppDomain, value: AppDomain.CurrentDomain.IsFullyTrusted),
                 new(ConfigTelemetryData.TraceMethods, value: settings.TraceMethods),
                 new(ConfigTelemetryData.ActivityListenerEnabled, value: settings.IsActivityListenerEnabled),
+                new(ConfigTelemetryData.ProfilerLoaded, value: Profiler.Instance.Status.IsProfilerReady),
+                new(ConfigTelemetryData.CodeHotspotsEnabled, value: Profiler.Instance.ContextTracker.IsEnabled),
```

Comment:
Do we have to use the static `Profiler.Instance`, this class is currently pretty testable 😿  I guess it doesn't really matter, but still

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/2733/commits/d99bd1a11751f6a1b3a3d39712c910bb76a97a4e